### PR TITLE
Fixes login issue

### PR DESCRIPTION
**Pull request by TheScienceOfSciences**
Login working with access to portfolio things:

Started with changes made anthonykrivonos

Changes made:

1. In Robinhood/endpoints.py, change the login method to return "https://api.robinhood.com/oauth2/token/".

2. Change the payload dict in the login function in Robinhood/Robinhood.py to:
payload = { 'password': self.password, 'username': self.username, "grant_type": "password", "client_id": "c82SH0WZOsabOXGP2sxqcj34FxkvfnWRZBKlBjFS" }

3. Change the last if statement in the login function in Robinhood/Robinhood.py to:

`        if 'access_token' in data.keys():
            self.auth_token = data['access_token']
            self.headers['Authorization'] = 'Bearer ' + self.auth_token
            return True`

### CODE REVIEW

File: Robinhood/Robinhood.py [hunk 1]
```diff
@@ -102,10 +102,7 @@ def login(self,
 
         self.username = username
         self.password = password
-        payload = {
-            'password': self.password,
-            'username': self.username
-        }
+        payload = { 'password': self.password, 'username': self.username, "grant_type": "password", "client_id": "c82SH0WZOsabOXGP2sxqcj34FxkvfnWRZBKlBjFS" }
```
Comment (vargasa) [inline Robinhood/Robinhood.py]: Regarding client_id. Check [this comment](https://github.com/Jamonek/Robinhood/pull/150#discussion_r232499886) out

File: Robinhood/endpoints.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 def login():
-    return "https://api.robinhood.com/api-token-auth/"
+    return "https://api.robinhood.com/oauth2/token/"
```
Comment (vargasa) [inline Robinhood/endpoints.py]: IMO would be good to define something like https://github.com/Jamonek/Robinhood/pull/150/files#diff-b9ce533de4140ddc89a86a86547eb17bR1

File: Robinhood/Robinhood.py (hunk 1 above)
Comment (TheScienceOfSciences) [reply Robinhood/Robinhood.py]: I will. Thanks!

File: Robinhood/endpoints.py (hunk 2 above)
Comment (TheScienceOfSciences) [reply Robinhood/endpoints.py]: I completely agree! Thanks!

File: Robinhood/Robinhood.py (hunk 1 above)
Comment (TheScienceOfSciences) [reply Robinhood/Robinhood.py]: @vargasa It looked to me like the mysterious client_id was required as part of the payload posted to login. Were you able to login without first posting the client_id?

**Comment by vargasa** [comment]
> Were you able to login without first posting the client_id?

I didn't try without it but it is send over when the Robinhood frontend 
ask for a token to the API. This is also certainly required to revoke 
the token.

**Comment by Jamonek** [comment]
Any updates?

**Pull request closed without merging**